activerecord/test/cases/persistence_test.rb

```diff
@@ -117,6 +117,15 @@ def test_increment_attribute
     assert_equal 53, accounts(:signals37, :reload).credit_limit
   end
 
+  def test_increment_aliased_attribute
```

Comment:
This test should also be moved to counter_cache test along with the related code change.